### DISCUSSION TITLE
Kobo Glo freezing

### DISCUSSION THREAD

**Discussion by defdefred** [Quill-OS/quill#57] [Q&A]
Hello, I have a kobo glo with freezing issue.
Suspecting the sd card to be defective, I tried InkBox with a new sd card.
All is booting perfectly, but after few ebook pages read it freeze again :-(

Could it be another hardware issue?
Is it possible to check the memory of the reader with InkBox? Like memtest86 tool?
After if memory is found defective, could it be possible to exclude part of the memory at boot time?

**Comment by Szybet** [comment]
was it like that on the original OS?

Maybe try to narrow down the issue, try to use the ereader for other things, try some apps, idk
Disable xorg for sure

It could be possible, it couls be possible - very unlikely
new SD card won't propably help

**Reply by defdefred** [reply]
Yes also freezing with original OS, this is the reason I tried InkBox on a new sd card.
Maybe i could see the error causing the freeze looking at the serial console?

**Reply by Szybet** [reply]
Well maybe. Depends if the kernel is still alive
